QuickBytes/src/main/java/com/quickbytes/model/Combo.java

```diff
@@ -1,20 +1,74 @@
-package com.quickbytes.model;
-
-import java.util.List;
+package com.springboot.backend.model;
 
+import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
-import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
 @Entity
+@Table(name= "combos")
 public class Combo {
-
 	@Id
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private Long comboId;
 	
-	@OneToMany //fk
-	private List<RequestCombo> requestCombos;
-}
```

Comment:
add this to your code for all of the OneToMany calls, there are a few that are in the main branch